setup.cfg

```diff
@@ -1,2 +1,71 @@
 [aliases]
-test=pytest
+test = pytest
```

Comment:
I recommend removing references to `setup.py test`. It's been deprecated for years; no one should be running `setup.py test`.